src/primitives/transaction.h

```diff
@@ -381,19 +382,21 @@ class CTxCeasedSidechainWithdrawalInput
         READWRITE(nullifier);
         READWRITE(pubKeyHash);
         READWRITE(scProof);
+        READWRITE(actCertData);
```

Comment:
let's add the comment that it is valid to have `actCertData` FE backed by empty vector.
Will be useful for SDK to remember, because in other places with FEs we consider always to have a full FE bytes.

Reply:
Done